azerothcore/azerothcore-wotlk
# Fix(DB/BWL): Trash Formation

## Changes Proposed:
-  Makes a formation with the goblins pack just after Broodlord Lashlayer, so they all come together like they should

## Issues Addressed:

- Partially fixes https://github.com/azerothcore/azerothcore-wotlk/issues/11523 (second part)

## SOURCE:


## Tests Performed:

- not tested



## How to Test the Changes:


1.  Go to the pack after broodlord lashlayer
2. pull them and see if they all come

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
